File: gcd/gcd.java
package gcd;

import java.util.Scanner;

class GCD {

    public static int findGCD(int a, int b) {
        while (a != 0) {
            int temp = b;
            b = a;
            a = temp % b;
        }
        return b;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter two integers: ");
        int num1 = scanner.nextInt();
        int num2 = scanner.nextInt();

        int gcd = findGCD(num1, num2);
        System.out.println("The GCD of " + num1 + " and " + num2 + ": " + gcd);

        scanner.close();
    }
}

File: gcd/gcd.c
#include <stdio.h>

int gcd(int a, int b) {
    while (b != 0) {
        int temp = b;
        b = a % b;
        a = temp;
    }
    return a;
}

int main() {
    int num1, num2;
    printf("Enter two integers: ");
    scanf("%d %d", &num1, &num2);

    int res = gcd(num1, num2);
    printf("The GCD of %d and %d: %d\n", num1, num2, res);

    return 0;
}
